en/core-libraries/internationalization-and-localization.rst

```diff
@@ -632,3 +632,8 @@ option to restrict which locales will automatically be used.
 .. meta::
     :title lang=en: Internationalization & Localization
     :keywords lang=en: internationalization localization,internationalization and localization,language application,gettext,l10n,pot,i18n,translation,languages
+    
+Translate Content/Entities
+==========================
+
+If you want to translate content/entities then you should look at the :doc:`Translate Behavior </orm/behaviors/translate>`
```

Comment:
This should probably go _above_ the page's metadata information.

Reply:
I'm sorry but I dont know what you mean 😅

Reply:
The `.. meta::` directive belongs to the page, not the "*Automatically Choosing the Locale Based on Request Data*" section, so your new section should be placed above the directive.

Reply:
oh sh*t, yea that seems logical 😓